monitor.py

```diff
@@ -104,58 +104,6 @@ def __init__(self, _db):
         self.image_scaler = ImageScaler(self._db, self.image_mirrors.values())
         self.oclc_linked_data = LinkedDataCoverageProvider(self._db)
 
-
```

Comment:
This is very messy code that's not needed very often. I moved it into a standalone script so that it's less urgent we provide test coverage.